ci-operator/jobs/openshift/installer/openshift-installer-release-4.16-presubmits.yaml

```diff
@@ -6857,7 +6857,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-vsphere-ovn,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
This looks unintentional.

Reply:
ugh, `make jobs` is so mysterious. fixed I think? (I had to rebase to current master first though to avoid undoing unrelated changes elsewhere.)